vcd/datasource_vcd_nsxt_ipsec_vpn_tunnel.go

```diff
@@ -69,6 +84,11 @@ func datasourceVcdNsxtIpSecVpnTunnel() *schema.Resource {
 				Computed:    true,
 				Description: "Public IPv4 Address of the remote device terminating the VPN connection",
 			},
+			"remote_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Remote ID",
```

Comment:
```suggestion
				Description: "ID of the remote peer site",
```
If I understand the docs correctly

Reply:
Yes, this is a valid comment. changed.